api/envoy/extensions/wasm/v3/wasm.proto

```diff
@@ -52,6 +52,7 @@ message VmConfig {
   // The Wasm runtime type.
   // Available Wasm runtime types are registered as extensions. The following runtimes are included
   // in Envoy code base:
+  // [#extension-category: envoy.wasm.runtime]
```

Comment:
please move this to directly above the field that is being extended - i think in this case `string runtime`

Reply:
https://github.com/envoyproxy/envoy/pull/17078/commits/2175b3acded1a52492d3a89e05389c3591e63b13 done